rasa/cli/test.py

```diff
@@ -109,6 +110,10 @@ def run_core_test(args: argparse.Namespace) -> None:
     else:
         test_core_models(args.model, stories, output)
 
+    rasa.shared.utils.cli.print_info(
+        f"Failed stories written to '{output}/{FAILED_STORIES_FILE}'"
```

Comment:
```suggestion
        f"Failed stories written to '{os.path.join(output, FAILED_STORIES_FILE)}'"
```
(otherwise, this will create wrong paths on windows)